README.md

```diff
@@ -1,2 +1,7 @@
-# ruby-homework
-A repository I will use to solve my Ruby homework
+# ruby-homework-modified
+A repository I will use to solve my Ruby homework from ruby course [https://github.com/monorkin/learn.rb](URL)
+
+##My favorite programming languages
+*C#*, **python**, *java*
```

Comment:
Take a look at [https://guides.github.com/features/mastering-markdown/](GitHub markdown guide) to see how lists are made. It's quite easy - you can just put `*` in front of the list item and place them in separate lines.